modules/ROOT/pages/telemetry-exporter.adoc

```diff
@@ -0,0 +1,146 @@
+
+=== Edit a Connection
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=connectionPermission]
+
+. Click the *Connections* tab.
+. Next to the connection you want to edit, click the *...* menu and select *Edit Connection*.
++
+The *Edit Connection* interface appears, showing the *Connection Details* page. 
+
+. In the *Connection Details* page, edit each field you want to change, and then click *Next.*
+. In the *Authentication* page, edit each field you want to change, then click *Test Connection*. When the connection is successful, click *Save Connection*.
+
+=== Delete a Connection
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=connectionPermission]
+
+. Next to the connection you want to delete, click the *...* menu and select *Delete Connection*.
+. A warning appears:
+* If the connection is not in use, the warning confirms you want to delete the connection. Click *Delete Connection*.
+* If the connection is in use, the warning states you must delete or redirect connections for any configurations that use the connection. Click *Dismiss*, and then <<edit-configuration,edit your configurations>> to use other existing connections before deleting the current connection.
+
+== Manage Configurations
+
+After you create a connection to a third-party app, you must create a _configuration_ to specify the scope of the audit logs to send to your third-party app. When you create a new configuration, you specify a business group and a destination. You can export audit logs from all of your business groups or a specific business group.
+// when traces return, add data type specification and environment
+You need the Telemetry Exporter Configuration Manager permission to manage configurations.
+// add data type when traces return
+
+include::partial$telemetry-exporter-notes.adoc[tag=hourDelay]
+
+=== Create a Configuration
+
+include::partial$include-nav-steps-telemetry-permissions.adoc[tag=configurationPermission]
+
+. Click *New Configuration*.
++
+The *Create New Configuration* interface appears, showing the *General Information* page.
+. In the *General Information* page, enter the following information:
+* *Configuration Name*: A name for your configuration using only alphanumeric characters.
+* *Connection*: An existing connection that you configured.
+// * *Data Type*: Audit logs or trace data
+. Click *Next*.
++
+The *Business Group & Environment Type* page appears.
+. In the *Select Business Group* section, select one of the following:
+* *All business groups*: Audit logs are sourced from all business groups in the organization.
+* *Select a business group*: Audit logs are sourced only from the business group you select.
+// audit logs are sourced -> data of the type you specified is sourced
+. Click *Next*.
+. Click *Save Configuration*. 
+
+////
+before the final *next*, add:
+. If you selected trace data as the data type, in the *Select Environment Type* section, select one of the following:
```

Comment:
Can you just say "If you selected trace data as the data type,  select the environment type"?

Reply:
```suggestion
. If you selected trace data as the data type, select one of the following:
```